firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -610,6 +610,7 @@ private boolean processActivatePutTask(Task<ConfigContainer> putTask) {
       // values from the put task must be non-null.
       if (putTask.getResult() != null) {
         updateAbtWithActivatedExperiments(putTask.getResult().getAbtExperiments());
+        rolloutsStateSubscriptionsHandler.publishActiveRolloutsState(putTask.getResult());
```

Comment:
What do you think about creating a new var from ` putTask.getResult()` called something like `activatedConfigs` above line 611 so we just need to call it once and then reference it in the rest of the `if` statement

Reply:
Oh yeah, that sounds good, will do that